CONTRIBUTING.md

```diff
@@ -58,6 +58,49 @@ module.exports = function ModuleName(options,UI) {
 }
 ```
 
+Modules can be distributed independently via npm, yarn and cdn. Any Independent Module Must follow this basic format
+
+```js
+function ModuleName(options,UI) {
+
+  var output;
+
+  function draw(input,callback) {
+
+    var output = function(input){
+      /* do something with the input */
+      return input;
+    }
+
+    this.output = output(input); // run the output and assign it to this.output
+    callback();
+  }
+
+  return {
+    options: options,
+    draw: draw,
+    output: output,
+    UI: UI
+  };
+}
+<!-- Info can be defined here or imported from a json file -->
+var info = {
+      "name": "ModuleName",
+      "description": "",
+      "inputs": {
+        // inputs here
+      };
+
+if(this.module) {
+  module.exports = [ModuleName,info];
+} else {
+  sequencer['ModuleName'] = {
```

Comment:
Is there a way to make this simpler, like could we have the sequencer produce a global called `module.exports` and then read it, so it's similar? Just brainstorming...

Reply:
oh @jywarren Actually I am doing that!! Please see the invert module